Symfony/CS/Tests/Fixer/Symfony/MethodSeparationFixerTest.php

```diff
@@ -204,7 +323,6 @@ public function method444f1()
     }
 
     /**/
-
```

Comment:
More tests added;
https://github.com/SpacePossum/PHP-CS-Fixer/commit/e4f96e19d41879b3045c0e0357a267246f3ab81d

I really cannot think of other cases to add, so please provide those if want. Please do so in PHP code.